File: domain/src/main/java/com/aceplus/domain/model/forApi/product/ProductCategory.java
package com.aceplus.domain.model.forApi.product;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by phonelin on 4/6/17.
 */

public class ProductCategory {

    @SerializedName("Id")
    @Expose
    String category_Id;

    @SerializedName("CategoryNo")
    @Expose
    String category_no;

    @SerializedName("name")
    @Expose
    String category_Name;

    public String getCategory_Id() {
        return category_Id;
    }

    public void setCategory_Id(String category_Id) {
        this.category_Id = category_Id;
    }

    public String getCategory_no() {
        return category_no;
    }

    public void setCategory_no(String category_no) {
        this.category_no = category_no;
    }

    public String getCategory_Name() {
        return category_Name;
    }

    public void setCategory_Name(String category_Name) {
        this.category_Name = category_Name;
    }


}

File: domain/src/main/java/com/aceplus/domain/entity/product/ProductCategory.kt
package com.aceplus.domain.entity.product

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName


@Entity(tableName = "product_category")
class ProductCategory {

    @PrimaryKey
    @ColumnInfo(name = "id")
    @SerializedName("id")
    @Expose
    var id: Int = 0

    @ColumnInfo(name = "category_id")
    @SerializedName("category_id")
    @Expose
    var category_id: String? = null

    @ColumnInfo(name = "category_name")
    @SerializedName("category_name")
    @Expose
    var category_name: String? = null

    @ColumnInfo(name = "category_no")
    @SerializedName("category_no")
    @Expose
    var category_no: String? = null


}